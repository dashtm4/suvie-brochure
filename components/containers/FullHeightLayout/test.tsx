import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { FullHeightLayout } from '.'

describe('<FullHeightLayout />', () => {
  it('should renders children', () => {
    const text = 'test'

    const wrapper = shallow(<FullHeightLayout>{text}</FullHeightLayout>)

    expect(wrapper.text()).toMatch(text)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })

  it('should render correct html', () => {
    const text = 'test'

    const wrapper = shallow(<FullHeightLayout>{text}</FullHeightLayout>)

    expect(wrapper.html()).toMatch(
      `<div class="min-h-screen flex flex-col">${text}</div>`
    )

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
