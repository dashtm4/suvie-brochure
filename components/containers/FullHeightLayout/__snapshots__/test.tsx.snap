// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FullHeightLayout /> should render correct html 1`] = `
<div
  className="min-h-screen flex flex-col"
>
  test
</div>
`;

exports[`<FullHeightLayout /> should renders children 1`] = `
<div
  className="min-h-screen flex flex-col"
>
  test
</div>
`;
